src/components/Sandbox.tsx

```diff
@@ -82,6 +82,7 @@ export class Sandbox extends React.Component<{}, {}>  {
     })(contentWindow.console.warn);
     contentWindow.console.error = (error => function() {
       logger.logLn(Array.prototype.join.call(arguments), "error");
+      logger.logLn(Array.prototype.join.call(["Failed to fetch:https://localhost:28843/out/main.wsm"]),"error");
```

Comment:
This will make all errors say "Failed to fetch:https://localhost:28843/out/main.wsm" no matter what has gone wrong, which is not what we want.

Reply:
ok I am working on it as you suggested.